src/Components/OATGraphViewer/OATGraphViewer.tsx

```diff
@@ -1101,6 +1113,7 @@ const OATGraphViewer = ({ state, dispatch }: OATGraphProps) => {
                             styles={buttonStyles}
                             onClick={onNewModelClick}
                             text={t('OATGraphViewer.newModel')}
+                            elementRef={newModelButtonRef}
```

Comment:
Rather than add another ref, could we get the element reference from the event args passed to onClick?